modules/ROOT/pages/constraints/managing-constraints.adoc

```diff
@@ -1029,24 +1058,166 @@ Node(0) with label `Actor` must have the properties (`firstname`, `surname`)
 
 ======
 
+[[modifying-property-constrained-property-will-fail]]
+==== Modifying type constrained properties will fail
+
+.Modify a type constrained property
+======
+
+.Modify the `title` for the `Movie` 'Iron Man' to an `INTEGER` value, given a constraint requiring `title` properties to be of type `STRING`
+[source, cypher, role=test-fail]
+----
+MATCH (m:Movie {title: 'Iron Man'})
+SET m.title = 13
+----
+
+.Error message
+[source, error]
+----
+Node(9) with label `Movie` required the property `title` to be of type `STRING`, but was of type `INTEGER`.
+----
+
+======
+
+
+[[constraint-creation-failures]]
+=== Constraint creation failures
+
+* xref:constraints/managing-constraints.adoc#create-an-already-existing-constraint[]
+* xref:constraints/managing-constraints.adoc#fail-to-create-constraint-due-to-existing-data[]
+* xref:constraints/managing-constraints.adoc#constraint-failures-and-indexes[]
+
+[[create-an-already-existing-constraint]]
+==== Creating an already existing constraint will fail
+
+Creating a constraint on the same label or relationship type and properties that are already constrained will fail.
```

Comment:
might also be nice to mention that the constraint type should be the same? (especially since we don't have the key and property uniqueness constraints on same schema in this group)